.circleci/config.yml

```diff
@@ -42,6 +42,20 @@ jobs:
     steps:
       - checkout
       - <<: *restore_dependency_cache
+      - run:
```

Comment:
It would be nice to cache this stuff so we're not making our builds even slower. This isn't a blocker (IMO), but something to think about instead.